hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -222,6 +222,10 @@ public final class HddsConfigKeys {
       "hdds.x509.ca.rotation.ack.timeout";
   public static final String HDDS_X509_CA_ROTATION_ACK_TIMEOUT_DEFAULT =
       "PT15M";
+  public static final String HDDS_X509_ROOTCA_CLIENT_POLLING_INTERVAL =
+      "hdds.x509.rootca.client.polling.interval";
```

Comment:
I would suggest change the name to "hdds.x509.rootca.certificate.polling.interval".